proto/backend.proto

```diff
@@ -64,6 +71,9 @@ message DataQueryRequest {
 
   // List of queries
   repeated DataQuery queries = 3;
+
+  //Info about the user who calls the plugin.
+  User user = 4;
```

Comment:
make sure we also add it to the resource call!

Reply:
https://github.com/grafana/grafana-plugin-sdk-go/pull/56/files#diff-ad0dcffdf701a0711096c6d383ad6cd5R105 :)